pull/BUILD.bazel

```diff
@@ -15,3 +15,32 @@ cc_library(
         "@civetweb",
     ],
 )
```

Comment:
besides `core` and `pull` there's also a `push` library.

Reply:
I added a commit to try to do something similar, but I can't seem to build some of the `push` targets. I get the following error:

```
ERROR: /home/palmer/.cache/bazel/_bazel_palmer/6d05ea7d43fb77a9b934c1b9ffb77807/external/com_github_whoshuu_cpr/BUILD.bazel:4:1: C++ compilation of rule '@com_github_whoshuu_cpr//:cpr' failed (Exit 1)
external/com_github_whoshuu_cpr/cpr/error.cpp:3:23: fatal error: curl/curl.h: No such file or directory
```

Is there possibly a dependency on curl that hasn't been managed by Bazel?

Reply:
yes, libcurl is expected to be installed on the system.